sdk/identity/azure-identity/src/test/java/com/azure/identity/UsernamePasswordCredentialTest.java

```diff
@@ -37,7 +37,7 @@ public class UsernamePasswordCredentialTest {
     public void testValidUserCredential() throws Exception {
         // setup
         String username = "testuser";
-        String password = "P@ssw0rd";
+        String password = "fakeCredentialPlaceholder";
```

Comment:
fakeCredentialPlaceholder -> fakeCredential ?

Reply:
Credscan will skip the check for the keyword `placeholder`. Use the string contains 'placeholder' in src code is better than suppress `fakeCredential` in CredScanSuppression file as it suppresses the specific place instead of every occurrence in repo (We have the guidance to suppress as specific as possible).